README.md

```diff
@@ -1,2 +1,22 @@
 # cre-contribs
 Repository for gathering various contributions/docs/snippets used by CRE
+
+### Exporters
+
+The exporter directory includes the necesary tooling to generate a basic kustomize template and vanilla kubernetes manifests for exporters. The exporters suported at the moment are:
+
+- promehteus-node-exporter
+- kube-state-metrics
+
+To build both exporters with the last helm charts version
```

Comment:
```
To build both exporters with the latest helm charts version
```

Reply:
Actually we pin the version now, so this description is out of date and needs to be changed.

```suggestion
To build both exporters with the pinned versions run the following command
```